pkgs/os-specific/linux/bolt/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, meson, ninja, pkgconfig, fetchFromGitLab,
+  python3, umockdev, gobject-introspection,
+  git, asciidoc, libxml2, libxslt, docbook_xml_dtd_45, docbook_xsl,
+  glib, systemd, polkit
+}:
+
+stdenv.mkDerivation rec {
+  name = "bolt-${version}";
+  version = "0.7";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "bolt";
+    repo = "bolt";
+    rev = "${version}";
+    sha256 = "0xn2c31kcjh1j76gq1qrcxwjyjyqnsxygkfrvh3xk07qc92f99xd";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkgconfig umockdev
+    git asciidoc libxml2 libxslt docbook_xml_dtd_45 docbook_xsl
+    (python3.withPackages
+      (p: [ p.pygobject3 p.dbus-python p.python-dbusmock ]))
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    glib systemd polkit
+  ];
+
+  postPatch = ''
+    echo '#!/bin/sh' > scripts/meson-install.sh
+    patchShebangs tests/test-integration
+  '';
+
+  mesonFlags = [
+    "-Dlocalstatedir=/var"
+  ];
+
+  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
+  PKG_CONFIG_UDEV_UDEVDIR = "${placeholder "out"}/lib/udev";
+
+  meta = with stdenv.lib; {
+    description = "Thunderbolt 3 device management daemon";
+    homepage = https://gitlab.freedesktop.org/bolt/bolt;
+    repositories.git = https://gitlab.freedesktop.org/bolt/bolt.git;
```

Comment:
Do we even use `meta.repositories` any more? In a very brief search I could only find that it was used by `nixpkgs-monitor` which is defunct now.

Reply:
Will remove.

I can't for the life of me figure out where I even found out about that attribute -- the manuals seem to be up to date. Must've been from stumbling across an old blog post or something.